### DISCUSSION TITLE
Complete the lyrics 🎤

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#605] [Galaxy Trivia] [answered]
```
When I'm ________ in my car
When a man come on the ________
He's ________ me more and more
About some ________ information
Supposed to fire my ________

I can't get ________
Oh no no no
Hey hey hey
That's what I ________

I can't get no ________
I can't get no ________
'Cause I try and I try and I try and I try
I can't get no, I can't get no
```

**Comment by peterbaccaro** [comment]
When I'm **DRIVING** in my car
When a man come on the **RADIO**
He's **TELLING** me more and more
About some **USELESS** information
Supposed to fire my **IMAGINATION**

I can't get **NO**
Oh no no no
Hey hey hey
That's what I **SAY**

I can't get no **SATISFACTION**
I can't get no **SATISFACTION**
'Cause I try and I try and I try and I try
I can't get no, I can't get no

**Comment by Veercodeprog** [comment]
The song is sung by the rolling stones frontman ,and one of my favorite song.

When I'm drivin' in my car, and that man comes on the radio;
And he's tellin' me more and more about some useless information,
Supposed to fire my imagination.
I can't get no, oh, no no, no, hey, hey, hey
That's what I say.

I can't get no satisfaction, I can't get no satisfaction,
'Cause I try and I try and I try and I try. I can't get no,
I can't get no.

**Comment by thuongtruong1009** [comment, marked as answer]
When I'm driving in my car
When a man come on the radio
He's telling me more and more
About
